mat_dp_pipeline/data_sources/mat_dp_db.py

```diff
@@ -3,10 +3,77 @@
 import pandas as pd
 
 from mat_dp_pipeline.abstract_data_sources import IndicatorsSource, IntensitiesSource
-from mat_dp_pipeline.data_sources.utils import location_to_path
+from mat_dp_pipeline.data_sources.country_sets import (
+    CountrySet,
+    CountrySets,
+    CustomCountry,
+    SourceWithCountries,
+)
 
+mat_dp_names_to_paths = {
+    "General": "/",
+    "North America": "/North America",
+    "Central and South America": "/Central and South America",
+    "CHL": "/Central and South America/Chile",
+    "Africa": "/Africa",
+    "Europe": "/Europe",
+    "Middle East and Central Asia": "/Middle East and Central Asia",
+    "South and East Asia": "/South and East Asia",
+    "Oceania": "/Oceania",
+    "NM": "/Africa/Namibia",
```

Comment:
Would be good to add a comment that NM isn;t a proper alpha-2, so we have to add it here.

Reply:
woops this one was actually a mistake